### DISCUSSION TITLE
Why is my account using github models showing "Your account has been blocked for suspected abuse. Please contact Support"?

### DISCUSSION THREAD

**Discussion by Roelauph** [community/community#139369] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

As the title suggests, I've never used github models.

**Comment by Chanduporalla** [comment]
If you're receiving the message "Your account has been blocked for suspected abuse" on GitHub, it could be due to a few possible reasons:

    Suspicious Activity: GitHub may block accounts if they detect suspicious or unusual behavior, such as excessive repository creation, spamming, or using bots that violate their terms of service.

    Automated Actions: Using automation tools or bots to perform actions on GitHub (e.g., creating issues, pull requests, or commenting) in a way that seems like abuse could trigger this response.

    Terms of Service Violation: If GitHub suspects that your account is involved in activities that go against their terms of service or community guidelines, such as spreading malware or violating copyrights, they might block it.

    Mass Reporting: Sometimes, accounts are blocked if they are reported by multiple users or flagged for inappropriate activity, even if this wasn't the user's intent.

Suggested Reply to GitHub Community:

You could respond to GitHub Support or the community in the following way:

Subject: Account Blocked for Suspected Abuse - Request for Assistance

Hi GitHub Support,

I recently received a notification that my account has been blocked for suspected abuse. I’m not aware of any actions on my part that would violate the GitHub Terms of Service or Community Guidelines.

Could you please provide more information about the issue and how I can resolve this? I would like to ensure that any unintentional mistakes are corrected, and I am willing to comply with GitHub’s rules.

Thank you for your help!

Best regards,
[Your Name]
[Your GitHub Username]

**Comment by anuxs** [comment]
+1 me too

**Reply by anuxs** [reply]
this is my turn：
>>Your access to Models/Copilot has been restricted because our systems detected the use of a credential sharing service which modifies and interferes with how Copilot works.

**Reply by h7ml** [reply]
+1

**Comment by KateCatlin** [comment, marked as answer]
Closing this issue as you can reach out to support for this [as described here](https://github.com/orgs/community/discussions/139369#discussioncomment-10717849)! Please reopen if there's something more we can do!
